sql/step_a10X_create_table_inv_and_invr.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE OH_MEDICALDSRINVENTORY (
+	MINVT_ID int NOT NULL AUTO_INCREMENT ,
+	MINVT_STATUS varchar (10)  NOT NULL ,
+	MINVT_DATE datetime NOT NULL ,
+	MINVT_US_ID_A varchar (50) NOT NULL,
```

Comment:
inconsistent formatting; sometime there is a space before the comma and other times there isn't.  FWIW, I prefer the lack of a space.